shared-ui/src/main/java/org/hzontal/shared_ui/bottomsheet/BottomSheetUtils.kt

```diff
@@ -97,52 +101,55 @@ BottomSheetUtils {
         consumer: LockOptionConsumer
```

Comment:
We need to align our formatting files so the whole file is not changing @djordjemarusic.